.github/actions/linting/action.yml

```diff
@@ -2,7 +2,7 @@ name: linting
 inputs:
   cwd:
     required: false
-    default: "${PWD}"
+    default: ${{ github.workspace }}
```

Comment:
Not sure why but ${PWD} did not work in many places but it does for the windows unit test action? Trying to use `${{ github.workspace }}` in windows breaks those builds.